Client/Frontend/Login/LoginListViewController.swift

```diff
@@ -194,7 +193,7 @@ extension LoginListViewController {
             let cell = tableView.dequeueReusableCell(withIdentifier: Constants.saveLoginsRowIdentifier, for: indexPath).then {
                 $0.textLabel?.text = Strings.saveLogins
                 $0.separatorInset = .zero
-                $0.accessoryView = toggle
+                $0.accessoryView =  !searchController.isActive ? toggle :  nil
```

Comment:
Maybe flip this condition so it check for `active` instead of `!active`?

Reply:
Fixed at https://github.com/brave/brave-ios/pull/4852/commits/6a414e58fbbb54d91b4409a5193d18714b7d5c7e